ui/src/common/components/OrcidProfileLink.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ExternalLink from './ExternalLink';
+
+function OrcidProfileLink({ children, orcid, className }) {
```

Comment:
OrcidProfileLinkOrChildren? something that makes it clearer that you can pass the children

Reply:
`OrcidProfileLinkOrChildren` indicates that it renders the link or the `children`, which is not the case the here.